gamemodes/terrortown/entities/weapons/weapon_ttt_unarmed.lua

```diff
@@ -71,18 +72,6 @@ function SWEP:Reload()
 
 end
 
----
--- @ignore
-function SWEP:Deploy()
-	if SERVER and IsValid(self:GetOwner()) then
-		self:GetOwner():DrawViewModel(false)
-	end
-
-	self:DrawShadow(false)
```

Comment:
This one had DrawShadow false too, can you please test, that all entities that did that really do not need to have this? 
Just make sure that none of them now still render a shadow / test ingame.